diasend.ts

```diff
@@ -123,3 +127,84 @@ export async function getPatientData(
 
   return response.data;
 }
+
+// SCRAPER for website
+const diasendWebsiteBaseUrl = "https://international.diasend.com/";
+
+export async function getAuthenticatedScrapingClient({
+  username,
+  password,
+  country = 108,
```

Comment:
108 is Germany, not sure what the country is used for but probably other values need to be supported as well